### ISSUE TITLE
01 -- Fly Your Flag -- Will Butts

### ISSUE THREAD

**Issue by Shea-Wolfe** [tiyd-python-2015-08/assigments#26]
- [x] Code It
  - [x] Draw a flag with one or more turtles
- [x] Press the Issue
  - [x] Open a _WIP issue_ named `01 -- Fly Your Flag -- YOUR NAME`
  - [x] Put this task list in the description
  - [x] Check off the boxes as you complete items
  - [x] Add a comment with the _Share_ link to your flag
- [x] Hard Mode (optional)
  - [x] Move flag drawing code to a function
  - [x] Add a paramter to the function called size
  - [x] Test your flag drawing at various sizes to ensure it works

**Comment by Shea-Wolfe** [comment]
This is my working flag.

https://trinket.io/python/a76148dddb

@powder-river 
@jamesmallen 

I'm taking your challenge to make the rising sun flag, but I wanted to get something in now.  I'm not sure I'm going to get there.  I feel like I made progress on coloring the outer stripes but for some reason I'm getting a name error when I try to run begin_poly() even though I found it in the documentation.  I hope to ask about it tomorrow.  I wasn't able to figure out how to stop at the edge of the flag, though I hope to look more at it tonight.  The code below is broken but I'm pasting it to save it.

https://trinket.io/python/40f1079353

**Comment by powder-river** [comment]
@Shea-Wolfe good use of mathematics and variables to create your flag. You'll be doing that a lot in this class.

**Comment by Shea-Wolfe** [comment]
@jamesmallen, @powder-river  

https://trinket.io/python/1ab3308808

I got there!  Ray colors still aren't working and the code is probably terribly overwritten but I can at least draw the shape as set out in the challenge.  I wanted to create each ray as a poly() which would (hopefully) fill in each after it was closed, leaving only a bit of white space in the corners but the begin_poly() function wasn't working for me.

Update:  Fixed the number of stripes (from 8 to 16)

https://trinket.io/python/8242078561

Final morning update:  I got it filled it!

https://trinket.io/python/079a659392
